roles/common/tasks/common.yml

```diff
@@ -28,12 +24,12 @@
   register: apt_key_change
   changed_when: apt_key_list.stdout != apt_key_change.stdout
 
-- name: Upgrade apt
-  apt:
-    upgrade: yes
-    update_cache: yes
-    cache_valid_time: "{{ apt_cache_valid_time }}"
-  when: "not travis_ci"
+# - name: Upgrade apt
```

Comment:
I don't believe we need this for now, it was working fine on a fresh box with this commented out. Any reason we should need it?